app/models/related-item.js

```diff
@@ -75,24 +78,6 @@ const Validations = buildValidations({
     }
     ),
   ],
-  relatedItemIdentifier: [
```

Comment:
I think this one was twofold. I first just thought it was easier to implement with an if clause like in the other cases but then I also saw that 20.1 is not mandatory. Although I do see the logic of including both attributes if you add one (if that was the intention).